nailgun/entities.py

```diff
@@ -138,6 +138,31 @@ def _check_for_value(field_name, field_values):
         )
 
 
+def _get_org_attrs(server_config, organization_label):
```

Comment:
i believe it is for the content view tests in robottelo. basically, it will help to get the organization's attributes from what is returned when creating a content view, if i am remembering it correctly.